zio-telemetry-example-proxy/src/main/scala/zio/telemetry/example/proxy/config/ServiceConfig.scala

```diff
@@ -0,0 +1,3 @@
+package zio.telemetry.example.proxy.config
+
+final case class ServiceConfig(url: String)
```

Comment:
Same here.

Reply:
I've tried but as I can see in PureConfig [documentation](https://pureconfig.github.io/docs/) it won't work properly. 

My intend here is to have configuration in the following format: 
```
service {
  url = "..."
}
``` 
And making it value class would expect format `url = "..."`. 

Check `final case class Port(value: Int) extends AnyVal` and reference [application.conf](https://github.com/pureconfig/pureconfig/blob/master/docs/src/main/resources/application.conf) from docs. 

If there's any way to do it, please let me know.

Reply:
See the comments.